__mocks__/react-native-permissions.ts

```diff
@@ -0,0 +1,76 @@
+import {PERMISSIONS, RESULTS} from 'react-native-permissions/dist/commonjs/permissions';
+import type {ValueOf} from 'type-fest';
+
+type Results = ValueOf<typeof RESULTS>;
+type ResultsCollection = Record<string, Results>;
+type NotificationSettings = Record<string, boolean>;
+
+const openLimitedPhotoLibraryPicker: jest.Mock<void> = jest.fn(() => {});
+const openSettings: jest.Mock<void> = jest.fn(() => {});
+const check: jest.Mock<Results> = jest.fn(() => RESULTS.GRANTED as string);
+const request: jest.Mock<Results> = jest.fn(() => RESULTS.GRANTED as string);
+const checkLocationAccuracy: jest.Mock<string> = jest.fn(() => 'full');
+const requestLocationAccuracy: jest.Mock<string> = jest.fn(() => 'full');
+
+const notificationOptions: string[] = ['alert', 'badge', 'sound', 'carPlay', 'criticalAlert', 'provisional'];
+
+const notificationSettings: NotificationSettings = {
+    alert: true,
+    badge: true,
+    sound: true,
+    carPlay: true,
+    criticalAlert: true,
+    provisional: true,
+    lockScreen: true,
+    notificationCenter: true,
+};
+
+const checkNotifications: jest.Mock<{status: Results; settings: typeof notificationSettings}> = jest.fn(() => ({
```

Comment:
Extract `{status: Results; settings: typeof notificationSettings}` to a separate type and use it here